.github/workflows/ui-tests-code-checks.yml

```diff
@@ -19,6 +19,11 @@ jobs:
       - name: Checkout the repository
         uses: actions/checkout@v2
 
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
```

Comment:
I guess we can use Node 16 and actions v3 here.

Reply:
We can use V3 and node 14 with npm 6 following these recommendations https://devdocs.prestashop-project.org/1.7/development/compile-assets/

Reply:
done